samples/client/petstore/lua/petstore/api/pet_api.lua

```diff
@@ -60,6 +60,10 @@ function pet_api:add_pet(pet)
 	--local var_content_type = { "application/json", "application/xml" }
 	req.headers:upsert("accept", "application/json")
 
+	-- TODO: create a function to select proper content-type
+	--local var_accept = { "application/xml", "application/json" }
+	req.headers:upsert("content-type", "application/xml")
```

Comment:
You're setting the content type to xml, but sending json.

Reply:
Right it's a known issue as mentioned in the code comment.
```
-- TODO: create a function to select proper content-type
```

As a workaround, users can rearrange the `accepts` to have `application/json` placed in the front.